src/flash/Flash.c

```diff
@@ -175,6 +175,9 @@ static size_t calculateBytesPerSector(const uint8_t *config) {
 }
 
 static size_t calculateBytesPerPage(const uint8_t *config) {
+    if (config[32] == 0x08) {
+        return 256;
+    }
```

Comment:
```suggestion
```

Shouldn't be required, since we don't support hybrid sector architecture.

> 64. Uniform 256-KB sectors = All sectors are uniform 256-KB with a 512B programming buffer.

-- [Datasheet  Page 154](https://www.mouser.de/datasheet/2/196/Infineon_S25FL128SS25FL256S_128_Mb__16_MB_256_Mb__-3166946.pdf)

Reply:
This function is supposed to set the correct page size, not the sector size. Pages can still have two different sizes, right? Either 256 or 512 bytes, even if all sectors are uniform 256 KB, or am I wrong somewhere?

Reply:
As far as I understand the datasheet the only possible page size for uniform Flash is set to 512.

Reply:
I'm not sure about that. How should we proceed?

![image](https://github.com/user-attachments/assets/d4f02cce-21ec-41c2-85c0-e385fa4354b4)

Reply:
I'll read through the datasheet more thoroughly and update you if I find something

Reply:
I think you are correct. I will change the function accordingly.